### PR TITLE
Align backing forms and view models #268

### PR DESCRIPTION
**Pull request by srikanthmalyala**
see #268

### CODE REVIEW

**Comment by srikanthmalyala** [comment]
@danfuterman  The PR is ready for review

File: src/backingform/BloodTestingRuleBackingForm.java [hunk 1]
```diff
@@ -45,7 +48,7 @@ public void setPattern(String pattern){
     }
     
     public void setContext(String context){
-        typingRule.setContext(BloodTestContext.valueOf(context));
+        typingRule.setContext(BloodTestContext.valueOf(context.replaceAll("\\s","").toUpperCase(Locale.FRENCH)));
```
Comment (danfuterman) [inline src/backingform/BloodTestingRuleBackingForm.java]: Is there a reason for using `Locale.FRENCH`? Also, why is it needed to remove all spaces? Are you just trying to do a `trim()`, or are you removing spaces between words?

Comment (srikanthmalyala) [inline src/backingform/BloodTestingRuleBackingForm.java]: Oh sorry  it's justt toUpperCase(), The backing form expects an enum value like "BLOODTYING" but the view model as "Blood Typing". so I'm trimming the spaces and making case upper

Comment (srikanthmalyala) [inline src/backingform/BloodTestingRuleBackingForm.java]: Not trimming , i'm removing the spaces

**Pull request merged**